src/SonarScanner.MSBuild.Common/Telemetry/PropertyProviderKind.cs

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * SonarScanner for .NET
+ * Copyright (C) 2016-2025 SonarSource SA
+ * mailto: info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarScanner.MSBuild.Common;
+
+public enum PropertyProviderKind
+{
+    CLI,
+    SONARQUBE_SCANNER_PARAMS,
+    SONARQUBE_ANALYSIS_XML,
+    SQ_SERVER_SETTINGS,
+    SONAR_SCANNER_OPTS, // Specifies options to the JVM for the sonar-scanner
```

Comment:
I think this was a mistake in the specification. `SONAR_SCANNER_OPTS` should never be the source of parameters as far as I know.
```suggestion
```

Reply:
Actually we read this in different places in the scanner for some settings.
Did not see it in depth but with a fast look it made sense.

I'll remove it and we can re-add it if there are any properties that we need telemetry for (did not check). It's easy.